src/approximations/importance.jl

```diff
@@ -13,7 +13,7 @@ convenient functions to generate samples and weights to approximate expectations
 - `rng`: random number generator objects
 - `nsamples`: number of samples generated by default
 """
-struct ImportanceSamplingApproximation{T, R}
+struct ImportanceSamplingApproximation{T, R} <: AbstractFormConstraint
```

Comment:
The `ImportanceSamplingApproximation` should not be declared as an `AbstractFormConstraint`. This is wrong from the semantic point of view. There should be an extra `MomentsApproximationFormConstraint` (or something similar).

Reply:
@ismailsenoz @bartvanerp